src/components/ContentResult.jsx

```diff
@@ -72,7 +74,8 @@ export const ContentResult = ({ contacts, allGroups }) => {
       searchValue
     )
     setFilteredContacts(filteredContactsBySearch)
-  }, [contacts, searchValue, selectedGroup, setFilteredContacts])
+    setContactsDisplayed(filteredContactsBySearch)
+  }, [contacts, searchValue, selectedGroup, setContactsDisplayed])
```

Comment:
What is the difference between filteredContacts and ContactsDisplayed?

Reply:
Aucune, si ce n'est que `ContactsDisplayed` est accessible pour tous les composants (`ContactsSelectionBar` aujourd'hui)

Reply:
Dans ce cas, pourquoi en créer un nouveau ? On devrait rendre accessible ce `ContactsDisplayed` partout, non ?

Reply:
`setFilteredContacts` est un état local déjà présent et surtout initialisé avec les contacts traités par son parent.
De part l'archi actuelle, le remplacer ici par le contexte `ContactsDisplayed`, fait apparaitre un instant le composant `ContactsEmptyList` (même problème en utilisant le contexte dans le parent)

Je n'ai pas de solution viable rapide, et je dois reprendre d'autres développement, mais à l'écoute si jamais !